src/xcode/ENA/ENA/Source/Services/Exposure Transaction/ExposureDetectionExecutor.swift

```diff
@@ -127,21 +161,3 @@ extension DownloadedPackagesStore {
 		}
 	}
 }
-
-private extension DownloadedPackagesStore {
-	func allPackages(for day: String, onlyHours: Bool) -> [SAPDownloadedPackage] {
```

Comment:
Just moved to where it belongs and removed `private` modifier so that it can be tested with our current structure.